solidity-plugin/src/main/java/org/sonarsource/solidity/SoliditySensor.java

```diff
@@ -128,8 +128,8 @@ private static void saveFileMeasures(SensorContext context, FileMeasures fileMea
     // String functionComplexityMeasure = fileMeasures.getFunctionComplexityDistribution().build();
     // context.<String>newMeasure().on(inputFile).withValue(functionComplexityMeasure).forMetric(CoreMetrics.FUNCTION_COMPLEXITY_DISTRIBUTION).save();
     //
-    String fileComplexityMeasure = Integer.toString(fileMeasures.getFileCognitiveComplexity());
-    context.<String>newMeasure().on(inputFile).withValue(fileComplexityMeasure).forMetric(CoreMetrics.FILE_COMPLEXITY_DISTRIBUTION).save();
+    // String fileComplexityMeasure = Integer.toString(fileMeasures.getFileCognitiveComplexity());
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=21&issues=AWQn3a8fYwZU9vvrp9dS&open=AWQn3a8fYwZU9vvrp9dS)

[ISSUE_KEY]: AWQn3a8fYwZU9vvrp9dS